### PR TITLE
Update docs to remove old Ansible content

### PR DESCRIPTION
**Pull request by ajacocks**

### CODE REVIEW

File: content/workshops/selinux_policy/exercise1.0.adoc [hunk 1]
```diff
@@ -1,220 +1,32 @@
...
-= Exercise 1.0 - Running Ad Hoc Commands
+= Exercise 1.0 - Introduction to Security Enhanced Linux
 
+== History
 
-== Exercise Description
+In this exercise, we are going to introduce you to Security Enhanced Linux, commonly known as `SELinux`.
 
-For our first exercise, we are going to run some ad hoc commands to help you get a feel for how Red Hat Ansible Tower works.  Ansible ad hoc commands enable you to perform repeatable tasks on remote nodes, without having to write a playbook.  They are very useful when you simply need to do one or two tasks quickly and often, to many remote nodes.
+SELinux is an implementation of the link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/4/html/SELinux_Guide/rhlcommon-appendix-0005.html#FTN.AEN5225[Flask] system security architecture.  The Flask architecture implements MAC, which focuses on providing an administratively-defined security policy that can control all subjects and objects, basing decisions on all security-relevant information. In addition, Flask focuses on the concept of least privilege, which gives a process exactly the rights it needs to perform it's given task.
```
Comment (Tekwriter1221) [inline content/workshops/selinux_policy/exercise1.0.adoc]: its given task

File: content/workshops/selinux_policy/exercise1.0.adoc [hunk 2]
```diff
@@ -1,220 +1,32 @@
...
 
+The Flask model allows you to express a security policy in a naturally flowing manner, so that parts of the security rules are like parts in a sentence. In Flask, changes are supported so you can tune your policy. Added to this architecture in the security server are TE and RBAC security models, providing fine-grained controls that can be transparent to users and applications.
```
Comment (Tekwriter1221) [inline content/workshops/selinux_policy/exercise1.0.adoc]: Spell out TE and RBAC on first use.

File: content/workshops/selinux_policy/exercise1.1.adoc [hunk 3]
```diff
@@ -1,170 +1,177 @@
...
 [source,bash]
 ----
----
-- hosts: web
-  name: Install the apache web service
-  become: yes
+Enforcing
 ----
 
+=== Step 3: Changing modes for specific domains
```
Comment (Tekwriter1221) [inline content/workshops/selinux_policy/exercise1.1.adoc]: there are two "Step 3"s

File: content/workshops/selinux_policy/exercise1.1.adoc [hunk 4]
```diff
@@ -1,170 +1,177 @@
...
+[source,bash]
+-----
+----
+time->Mon Nov 17 00:45:36 2008
+type=SYSCALL msg=audit(1226882736.442:86): arch=40000003 syscall=196 success=no exit=-13 a0=b9a1e198 a1=bfc2921c a2=54dff4 a3=2008171 items=0 ppid=2425 pid=2427 auid=502 uid=48 gid=48 euid=48 suid=48 fsuid=48 egid=48 sgid=48 fsgid=48 tty=(none) ses=4 comm="httpd" exe="/usr/sbin/httpd" subj=unconfined_u:system_r:httpd_t:s0 key=(null)
+type=AVC msg=audit(1226882736.442:86): avc:  denied  { getattr } for  pid=2427 comm="httpd" path="/var/www/html/file1" dev=dm-0 ino=284133 scontext=unconfined_u:system_r:httpd_t:s0 tcontext=unconfined_u:object_r:samba_share_t:s0 tclass=file
+-----
 
-Use the `write/quit` method in `vi` or `vim` to save your playbook, i.e. `Esc :wq!`
+Here, we can see that the `httpd` process, ID `2427`, was disallowd from checking attributes of the file `/var/www/html/file1`.  Additionally, we can see that the domain of the process was `httpd_t`, and the file was labeled with the `samba_share_t` context.
```
Comment (Tekwriter1221) [inline content/workshops/selinux_policy/exercise1.1.adoc]: disallowed

File: content/workshops/selinux_policy/exercise1.1.adoc (hunk 3 above)
Comment (ajacocks) [reply content/workshops/selinux_policy/exercise1.1.adoc]: Fixed.

File: content/workshops/selinux_policy/exercise1.0.adoc (hunk 2 above)
Comment (ajacocks) [reply content/workshops/selinux_policy/exercise1.0.adoc]: Fixed.

File: content/workshops/selinux_policy/exercise1.0.adoc (hunk 1 above)
Comment (ajacocks) [reply content/workshops/selinux_policy/exercise1.0.adoc]: Fixed.

File: content/workshops/selinux_policy/exercise1.1.adoc (hunk 4 above)
Comment (ajacocks) [reply content/workshops/selinux_policy/exercise1.1.adoc]: Fixed.

**Pull request merged**